src/operations/extrusions/extrudeRotate.js

```diff
@@ -103,11 +103,11 @@ const extrudeRotate = (options, geometry) => {
   const baseSlice = slice.fromSides(geom2.toSides(geometry))
   slice.reverse(baseSlice, baseSlice)
 
-  const createSlice = (t, index) => {
+  const createSlice = (percent, index, base) => {
```

Comment:
love it !
as mentionned above though, not sure if 'percent' or 'progress' is best ?